rskj-core/src/main/java/co/rsk/peg/whitelist/LockWhitelistEntry.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is part of RskJ
+ * Copyright (C) 2017 RSK Labs Ltd.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package co.rsk.peg.whitelist;
+
+import co.rsk.bitcoinj.core.Address;
+import co.rsk.bitcoinj.core.Coin;
+
+/**
+ * Represents a lock whitelist
+ * entry for a LockWhiteList.
+ *
+ * @author Jose Dahlquist
+ */
+public interface LockWhitelistEntry {
```

Comment:
I'd add a method to check whether a certain amount can be locked against this, and then use it to validate on the whitelist side.